### PR TITLE
Created HomeScreen/QuickAction

### PR DESCRIPTION
**Pull request by sksinha2410**
### Description:
- Created `QuickActionBar` for `HomeScreen`.
- It contains four options: `Send`, `Receive`, `Invoice` and `History`.
- It has `transparent` background with `Black` fonts and buttons.
- Added `VectorIcon` library. 
 
 ## Vector Icons
- To use Vector icons : ``` import Icon from 'react-native-vector-icons/MaterialIcons'; ```
- List of available vector icons: https://oblador.github.io/react-native-vector-icons/#MaterialIcons

## Sample code to use:
 ```
import React from 'react';
import { SafeAreaView, StyleSheet, StatusBar } from 'react-native';
import QuickActionBar from './src/UI/Components/QuickAction/index';

const App: React.FC = () => {
    return (
        <SafeAreaView style={styles.container}>
            <StatusBar barStyle="dark-content" />
            <QuickActionBar />
        </SafeAreaView>
    );
};

const styles = StyleSheet.create({
    container: {
        flex: 1,
        backgroundColor: '#cba',
    },
});
export default App;
```

## Output Screenshoots:

Screenshot_20240703-112025~2

### CODE REVIEW

File: src/UI/Components/QuickAction/QuickAction.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+// QuickActionBar.tsx
+import React from 'react';
+import { View, Text, TouchableOpacity } from 'react-native';
+import AvatarCircle from '../AvatarCircle/index';
+import { styles } from './Style';
+
+const QuickAction: React.FC = () => {
```
Comment (Jha2508) [inline src/UI/Components/QuickAction/QuickAction.tsx]:
```suggestion
const QuickAction = () => {
```

Comment (Jha2508) [reply src/UI/Components/QuickAction/QuickAction.tsx]: +docs

**Comment by Jha2508** [comment]
try using icons from some library instead, do not upload all icons to assets directly

**Comment by sksinha2410** [comment]
> try using icons from some library instead, do not upload all icons to assets directly

Not sure  how to use icon library, will take care next time. Remain comments has been addressed.

**Comment by Jha2508** [comment]
That is not the way you do development. The reason I pushed you to use icons from library is because we are increasing app size by each and every image we have in assets. Please remove the assets and use the icons from the library. Also, why is gencode pushed?

**Comment by sksinha2410** [comment]
> That is not the way you do development. The reason I pushed you to use icons from library is because we are increasing app size by each and every image we have in assets. Please remove the assets and use the icons from the library. Also, why is gencode pushed?

Okay! I will do that. 

Actually I have pull `develop` branch to my branch, that was the only commit I was behind of, resulting that file in commit. I haven't changed anything in that file.

**Comment by Jha2508** [comment]
@sksinha2410 What is the update on the PR? Let us get this PR merged today once you use the icons library. Also, check React Native Paper if we can use it

**Comment by sksinha2410** [comment]
> @sksinha2410 What is the update on the PR? Let us get this PR merged today once you use the icons library. Also, check React Native Paper if we can use it

PR is updated. Now vector icon library is being used instead of images.

**Review by Jha2508 — APPROVED** [review]

**Pull request merged**